department-of-veterans-affairs/caseflow
# Div/appeals 44412

Resolves [Correspondence incorrect name and location for vbms doc types.json](https://jira.devops.va.gov/browse/APPEALS-44412)

# Description
Renamed the filename which has spaces in it. Also pushed, the file in the mentioned folder.

## Acceptance Criteria

## Testing Plan

1. Log into caseflow as [[INBOUND_OPS_TEAM_ADMIN_USER_S2]
2. Click on Your Queue
3. Click on the Switch views button and select Correspondence Cases
4. Select any veteran having NOD package document type of review package.
5. From review package page, click on Edit link at Document type and check for option _New document type_ is loading with all the preferred data.


# Frontend
## User Facing Changes

 BEFORE|AFTER
<img width="620" alt="Screenshot 2024-06-11 at 2 14 25 PM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/135847343/492eebef-1975-4b48-b5f9-a12bfd85717a">


## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
